opensearch-operator/api/v1/opensearch_types.go

```diff
@@ -48,7 +48,8 @@ type GeneralConfig struct {
 	// Additional volumes to mount to all pods in the cluster
 	AdditionalVolumes []AdditionalVolume `json:"additionalVolumes,omitempty"`
 	// Populate opensearch keystore before startup
-	Keystore []KeystoreValue `json:"keystore,omitempty"`
+	Keystore []KeystoreValue  `json:"keystore,omitempty"`
+	Snapshot []SnapshotConfig `json:"snapshot,omitempty"`
```

Comment:
This should be named something like `SnapshotRepositories`, otherwise its confusing.

Reply:
Thanks @swoehrl-mw I have renamed to `SnapshotRepositories`, should be as follows.
```
  general:
    snapshotRepositories: 
       - name: my_s3_repository_3
         type: s3
         settings:
          bucket: opensearch-s3-snapshot
          region: us-east-1
          base_path: os-snapshot_3
 ```